docs/lib/auth/working-with-api.md

```diff
@@ -1,7 +1,14 @@
 ---
-title: Working with API
+title: Working with other Category
```

Comment:
This should be other Categories but I also don't think this matches the content you have below. Working with other categories is just as simple as making sure Auth is added and the user is signed in. Maybe this section should be removed and this topic covered under the Escape Hatch section. Either that or it should be renamed to something like, "Getting credentials" where we explain that this is not something the user should need to worry about or do at all within Amplify but if they really want to get them, they can use the escape hatch method by casting fetchAuthSession.